src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -415,6 +411,92 @@ MatrixHeader.contextTypes = {
     navigate: PropTypes.func,
 };
 
+const ClickableThumbnail = (props) => {
+    const expandedThumbnail = props.expandedThumbnail;
+    const expandThumbnail = props.expandThumbnail;
+    return (
+        <button
+            className={`body-image-thumbnail ${expandedThumbnail ? 'expanded' : 'collapsed'}`}
+            onClick={() => expandThumbnail()}
+        >
+            <div className="body-map-expander">Filter results by body diagram</div>
+            {svgIcon('expandArrows')}
+            <BodyDiagram />
+            <div className="body-list body-list-narrow">
+                <ul className="body-list-inner">
+                    {Object.keys(CellsList).map(image =>
+                        <div
+                            className={`body-inset ${image}`}
+                            id={image}
+                            key={image}
+                        >
+                            <img className="active-image" src={`/static/img/bodyMap/insetSVGs/${image.replace(' ', '_')}.svg`} alt={image} />
+                            <img className="inactive-image" src={`/static/img/bodyMap/insetSVGs/${image.replace(' ', '_')}_deselected.svg`} alt={image} />
+                            <div className="overlay" />
+                        </div>
+                    )}
+                </ul>
+            </div>
+        </button>
+    );
+};
+
+ClickableThumbnail.propTypes = {
+    expandedThumbnail: PropTypes.bool.isRequired,
+    expandThumbnail: PropTypes.func.isRequired,
+};
+
+const SelectedFilters = (props) => {
+    const selectedFilters = props.filters;
+    const organTerms = selectedFilters.filter(f => f.field === organField);
+    const systemsTerms = selectedFilters.filter(f => f.field === systemsField);
+    const freeSearchTerms = selectedFilters.filter(f => f.field === 'searchTerm');
+    const selectedTerms = [...organTerms, ...systemsTerms, ...freeSearchTerms];
+    return (
+        <React.Fragment>
```

Comment:
<React.Fragment> or <> is fine.

Reply:
I think I talked to @forresttanaka about this at some point and I don't remember exactly what we discussed, but I think we're mostly sticking to <React.Fragment> except for specific cases?

Reply:
We don’t have a style guide for this, and I already discussed this with Phil a while back. I think `<>` on its own line is ugly so I prefer `<React.Fragment>` for those cases, and I prefer `<>` when inline. But I don’t comment about it in code reviews because we have no guidance on it — whatever people like.